### ISSUE TITLE
Doesn't work with AGP 2.4.0-alpha7

### ISSUE THREAD

**Issue by yogurtearl** [aurae/android-junit5#7]
This plugin seems to have problems with Android Gradle Plugin (AGP) 2.4.0-alpha7 (works fine with 2.3.2)

With AGP 2.4.0-alpha7 I get this error:

Looks like the test engines are not on the classpath for the ConsoleLauncher invocation.

```:app:junitPlatformTestDebug
org.junit.platform.commons.util.PreconditionViolationException: Cannot create Launcher without at least one TestEngine; consider adding an engine implementation JAR to the classpath
        at org.junit.platform.commons.util.Preconditions.condition(Preconditions.java:285)
        at org.junit.platform.launcher.core.DefaultLauncher.<init>(DefaultLauncher.java:52)
        at org.junit.platform.launcher.core.LauncherFactory.create(LauncherFactory.java:53)
        at org.junit.platform.console.tasks.ConsoleTestExecutor.executeTests(ConsoleTestExecutor.java:61)
        at org.junit.platform.console.tasks.ConsoleTestExecutor.lambda$execute$0(ConsoleTestExecutor.java:57)
        at org.junit.platform.console.tasks.CustomContextClassLoaderExecutor.invoke(CustomContextClassLoaderExecutor.java:33)
        at org.junit.platform.console.tasks.ConsoleTestExecutor.execute(ConsoleTestExecutor.java:57)
        at org.junit.platform.console.ConsoleLauncher.executeTests(ConsoleLauncher.java:79)
        at org.junit.platform.console.ConsoleLauncher.execute(ConsoleLauncher.java:69)
        at org.junit.platform.console.ConsoleLauncher.main(ConsoleLauncher.java:41)

Option                                        Description
------                                        -----------
-h, --help                                    Display help information.
--disable-ansi-colors                         Disable ANSI colors in output (not
                                                supported by all terminals).```

**Comment by aurae** [comment]
This is odd; Are you using version `1.0.0-M4` of the plugin? I remember having this issue with the M3 release, possibly because of some recent changes in how Android Studio populates the classpath for annotation processors. It went away after updating everything to M4, though.

Could you also try adding the required TestEngine manually, like so:

```groovy
testApk ""org.junit.jupiter:junit-jupiter-engine:5.0.0-M4"
```

**Comment by aurae** [comment]
I have just released version `1.0.0-M4-rev1` of the plugin, which is supposed to address this issue. Please let me know if it doesn't!
